### PR TITLE
acrobat-reader-dc

### PR DESCRIPTION
**Pull request by yurikoles**
For now I created the separate cask, but may be I need to update adobe-reader one?

### CODE REVIEW

**Comment by yurikoles** [comment]
If we update adobe-reader, then we can purge all localized versions, since this version is multilingual

File: Casks/acrobat-reader-dc.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+						 'com.adobe.acrobat.DC.reader.appsupport.pkg.MUI',
+						 'com.adobe.acrobat.DC.reader.browser.pkg.MUI',
+						],
+  	   		:delete => '/Applications/Adobe Acrobat Reader DC.app'
+
+  zap       :delete => [
+  						'~/Library/Application Support/Adobe/Acrobat/DC',                   
+                        '~/Library/Preferences/com.adobe.Reader.plist',
+                	   ]
+end
```
Comment (vitorgalvao) [inline Casks/acrobat-reader-dc.rb]: It has a “no newline at end of file” warning. Those are [actually meaningful](http://unix.stackexchange.com/a/18789).

**Comment by vitorgalvao** [comment]
How is it multilingual? Do we pick the language, or does it follow the system language?

**Comment by yurikoles** [comment]
It picks system language and doesn't allow to change it.

**Comment by vitorgalvao** [comment]
Then this doesn’t make the other language casks obsolete. Many people like having the system in one language and apps in another. Is the regular (non-dc) reader still available, though?

**Comment by yurikoles** [comment]
So I have added new line at end.

**Comment by vitorgalvao** [comment]
> Is the regular (non-dc) reader still available, though? Because if it is, then this is simply an alternative version, and belongs in [caskroom/versions](https://github.com/caskroom/homebrew-versions).

**Comment by yurikoles** [comment]
Yes, they are available

**Comment by vitorgalvao** [comment]
Then we need this moved to the other repo, please.

**Pull request closed without merging**